PeasyExample/PeasyExample/ViewController.swift

```diff
@@ -15,7 +15,7 @@ final class ViewController: UIViewController {
 	
 	override func viewDidLoad() {
 		super.viewDidLoad()
-		server.start()
+		try! server.start()
```

Comment:
The force tries is really what I was hoping to avoid which is why I haven't got to this issue yet (already raised an issue for this in #1). It's really ugly to have to do this in tests compared to the nice `server.start()`. 

Totally appreciate they're basically doing the same thing, except with a try! you don't get the fatalError message, so harder to know exactly what's gone wrong?

What I was going to investigate was if we could have two start functions, one that throws and one that doesn't. They might need to be named differently, I'm just not sure exactly what yet.

Reply:
You can overload based on the throwingness but it's not recommended (previously it's caused an issue in JSONUtilities which we use https://github.com/lucianomarisi/JSONUtilities/issues/31)